packages/e2e-tests/src/specs/editor/fontCheck/fontCheck.js

```diff
@@ -141,7 +144,7 @@ describe('Font Check', () => {
     await expect(page).toMatchElement('h2', { text: 'Dashboard' });
   });
 
-  it('should redirect to dashboard when pressing ESC', async () => {
+  it('should redirect to dashboard when pressing ESC !', async () => {
```

Comment:
?

Reply:
Yup flagged above --- I should have flipped this PR back to draft 

https://github.com/GoogleForCreators/web-stories-wp/pull/10918#issuecomment-1072258228